sdk/plugins/host/options.go

```diff
@@ -1,13 +1,10 @@
 package external_host_plugins
 
 import (
-	"errors"
 	"fmt"
-	"io/fs"
-	"os"
 
-	pb "github.com/hashicorp/boundary/sdk/pbs/plugin"
 	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/go-secure-stdlib/pluginutil/v2"
```

Comment:
Reviewers: Same deal; this now supports pass-through options to pluginutil instead of hosting them directly.